### PR TITLE
Ag 3944 angular component types

### PR DESCRIPTION
**Pull request by StephenCooper**

### CODE REVIEW

File: community-modules/angular/gulpfile.js [hunk 1]
```diff
@@ -32,38 +30,13 @@ const mainPostCompileRename = () => {
...
 
-const updateProperties = (callback) => {
-    gridToNg.updatePropertiesSrc(callback);
-};
-
 const compileSource = (callback) => {
     return ngc('./tsconfig-src.json', callback);
 };
 
-const copyFromModuleSource = () => {
```
Comment (StephenCooper) [inline community-modules/angular/gulpfile.js:43]: copying is done from within the grid-packages so removing from here.

File: community-modules/angular/package.json [hunk 2]
```diff
@@ -53,7 +53,6 @@
     "ts-node": "~8.3.0",
     "tslint": "~5.18.0",
     "typescript": "~3.7.5",
-    "replace-in-file": "^4.1.0",
-    "typescript-simple": "8.0.6"
```
Comment (StephenCooper) [inline community-modules/angular/package.json:57]: No longer required after deleting watch task.

File: community-modules/core/src/ts/main.ts [hunk 3]
```diff
@@ -323,7 +323,8 @@ export {
     DndSourceCallbackParams,
     EditableCallbackParams,
     SuppressPasteCallbackParams,
```
Comment (StephenCooper) [inline community-modules/core/src/ts/main.ts:325]: Adding missing Interfaces as revealed by now typing angular component.

**Review by seanlandsman — APPROVED** [review]
Looks good to me, but you'll also need to change the Vue (not Vue 3) projects to use ComponentUtil.PUBLIC_EVENTS as they currently rely on ComponentUtil.EVENTS

Otherwise looks good

**Review by rmc-software — APPROVED** [review]

**Comment by StephenCooper** [comment]
> Looks good to me, but you'll also need to change the Vue (not Vue 3) projects to use ComponentUtil.PUBLIC_EVENTS as they currently rely on ComponentUtil.EVENTS
> 
> Otherwise looks good

I only see EVENTS being used in the Vue 3 module.

**Pull request merged**